.github/workflows/docker_publish.yml

```diff
@@ -50,54 +60,113 @@ jobs:
         with:
           file: CI/Dockerfile_0_base
           context: .
-          push: true
+          push: ${{ matrix.isMerged }}
           build-args: |
             UBUNTU_VERSION=${{ matrix.ubuntu_versions }}
-          tags: ghcr.io/svalinn/dagmc-ci-ubuntu-${{ matrix.ubuntu_versions }}
+          tags: ghcr.io/svalinn/test_dagmc-ci-ubuntu-${{ matrix.ubuntu_versions }}
```

Comment:
This will need to include some other unique string (sha?) to avoid two simultaneous PR's from overwriting each other